tests/crypto/prj.conf

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# TEST
+CONFIG_ZTEST=y
+CONFIG_ZTEST_STACKSIZE=65536
+CONFIG_MAIN_STACK_SIZE=8096
+CONFIG_TEST_USERSPACE=n
+CONFIG_ZTEST_TC_UTIL_USER_OVERRIDE=y
+
+# LOG
+CONFIG_LOG=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+CONFIG_LOG_IMMEDIATE=y
+CONFIG_LOG_DEFAULT_LEVEL=4
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_UART=n
+CONFIG_HAS_SEGGER_RTT=y
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG_BACKEND_RTT_MODE_BLOCK=y
+CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=16384
+CONFIG_LOG_BACKEND_SHOW_COLOR=n
+CONFIG_LOG_BACKEND_FORMAT_TIMESTAMP=n
+
+# MISC
+CONFIG_DK_LIBRARY=y
+CONFIG_FLOAT=y
+CONFIG_MAIN_STACK_SIZE=8096
+CONFIG_SPEED_OPTIMIZATIONS=y
```

Comment:
It should be commented here why we don't use the default -Os

Reply:
Optimization is now default in 7aee7a9 .
Might be used in the future if time measurements are used.